### PR TITLE
[RGAA] Add information aria label review item dashboard component

### PR DESCRIPTION
**Pull request by antonia2929**
Part of this trello card :  [add aria-label for revision dashboard "reviewable list"](https://trello.com/c/ojoyK0Z2/77-add-aria-label-for-revision-dashboard-reviewable-list)

**Detailed purpose of the PR**
Add an aria label on information icon in reviewable skills.

**Testing Strategy**
- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Review by StefanoA1 — APPROVED** [review]

File: packages/@coorpacademy-components/src/atom/review-presentation/index.js [hunk 1]
```diff
@@ -29,22 +29,28 @@ const getIcon = icon => {
...
+          aria-label={moreDetailsAriaLabel}
+        />
       </div>
       <div className={style.toolTip}>
-        <div className={style.tooltipText}>{tooltipText}</div>
+        <div className={style.tooltipText}>
+          <p>{tooltipText}</p>
+        </div>
       </div>
     </div>
```
Comment (DjamelSoualmi) [inline packages/@coorpacademy-components/src/atom/review-presentation/index.js:48]: tu peux supprimer une duv t'as pas besoin de deux 

```
   <div className={style.toolTip}>
        <p className={style.tooltipText}>
          {tooltipText}
        </p>
      </div>
```

File: packages/@coorpacademy-components/src/atom/review-presentation/index.js [hunk 2]
```diff
@@ -29,22 +29,28 @@ const getIcon = icon => {
   return <Icon className={style.labelIcon} />;
 };
 
-const ToolTip = ({tooltipText}) => {
+const ToolTip = ({tooltipText, moreDetailsAriaLabel}) => {
```
Comment (DjamelSoualmi) [inline packages/@coorpacademy-components/src/atom/review-presentation/index.js:32]:
```suggestion
const ToolTip = ({tooltipText, 'aria-label': moreDetailsAriaLabel}) => {
```

File: packages/@coorpacademy-components/src/atom/review-presentation/index.js [hunk 3]
```diff
@@ -65,7 +71,10 @@ const ReviewPresentation = props => {
                 <div className={style.reviewListText}>
                   {getIcon(key)} {label.text}
                 </div>
-                <ToolTip tooltipText={label.tooltipText} />
+                <ToolTip
+                  tooltipText={label.tooltipText}
+                  moreDetailsAriaLabel={label.moreDetailsAriaLabel}
```
Comment (DjamelSoualmi) [inline packages/@coorpacademy-components/src/atom/review-presentation/index.js:76]:
```suggestion
                 'aria-label'={label.moreDetailsAriaLabel}
```

File: packages/@coorpacademy-components/src/atom/review-presentation/index.js [hunk 4]
```diff
@@ -76,7 +85,8 @@ const ReviewPresentation = props => {
 };
 
 ToolTip.propTypes = {
-  tooltipText: PropTypes.string
+  tooltipText: PropTypes.string,
+  moreDetailsAriaLabel: PropTypes.string
```
Comment (DjamelSoualmi) [inline packages/@coorpacademy-components/src/atom/review-presentation/index.js:89]:
```suggestion
 'aria-label': PropTypes.string
```

**Review by DjamelSoualmi — CHANGES_REQUESTED** [review]

**Review by DjamelSoualmi — APPROVED** [review]

**Pull request merged**